common/views/components/ButtonSolidLink/ButtonSolidLink.tsx

```diff
@@ -12,9 +12,18 @@ import ConditionalWrapper from '../ConditionalWrapper/ConditionalWrapper';
 
 type ButtonSolidLinkProps = ButtonSolidBaseProps & {
   clickHandler?: (event: SyntheticEvent<HTMLAnchorElement>) => void;
-  link: {href: {pathname: string, query: string}, as: {pathname: string, query: string}} | string;
+  link:
+    | {
+        href: { pathname: string; query: string };
```

Comment:
Don't we have this typed somewhere else?

Reply:
We should do, yeah. I didn't immediately find it – will look again.